components/Input/input.css

```diff
@@ -0,0 +1,203 @@
+.input-grp {
+  margin: 0 2rem 1rem;
+  color: var(--color-grey-dark);
+  font-size: 1.6rem;
+}
+
+.input-grp > label {
+  display: block;
+  color: var(--color-grey-dark);
+}
+
+.input-grp.required > label::after {
+  content: " *";
+  color: inherit;
+}
+
+.input-grp.err > label,
+.input-grp .err-msg {
+  color: var(--color-error);
+}
+
+.input-grp.err > input {
+  border: 1px solid var(--color-error);
+}
+
+.input-grp.err > input:focus {
+  box-shadow: none;
+}
+
+.input-grp > input {
+  font-size: inherit;
+  outline: none;
+  padding: 0.8rem 1.3rem;
+  min-width: 30rem;
+  color: var(--color-grey-dark-3);
+  border: 1px solid var(--color-grey-dark-2);
+  border-radius: var(--border-radius-3);
+}
+
+.input-grp > input:focus {
+  box-shadow: 0 0 0 1px var(--color-primary);
+}
+
+.fancy > input {
+  background-color: var(--color-tertiary);
+  border: none;
+  border-bottom: 2px solid transparent;
+  display: block;
+  transition: all 0.3s;
+}
+
+.fancy > label {
+  font-weight: 700;
+  font-size: 1.4rem;
+  margin-left: 2rem;
+  margin-top: 0.7rem;
+  transition: all 0.3s;
+}
+
+.fancy.err > input {
+  border: none;
+  border-bottom: 2px solid var(--color-error) !important;
```

Comment:
I think you are using `!important` because the color changes when the input is active. You can add  `.fancy.err > input:focus` in here as well to get rid of that issue and then remove the `!important` as well